### PR TITLE
Remove the artifact_info file references in our integration tests

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

File: tests/tests/test_basic_integration.py [hunk 1]
```diff
@@ -156,22 +125,13 @@ def do_test_failed_updated_and_valid_update(
         )
 
     def do_test_update_no_compression(self, env, valid_image_with_mender_conf):
-        """Uploads an uncompressed artifact, and runs the whole update process."""
+        """Uploads an uncompressed artifact, and runs the whole udpate process."""
```
Comment (lluiscampos) [inline tests/tests/test_basic_integration.py:128]: This specific line would be nice to drop from the revert or re-aply in a new commit.

File: tests/tests/test_image_update_failures.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def do_test_update_image_id_already_installed(
             mender_conf = mender_device.run("cat /etc/mender/mender.conf")
             deployment_id, expected_image_id = common_update_procedure(
                 valid_image_with_mender_conf(mender_conf),
-                True,
+                verify_status=True,
```
Comment (lluiscampos) [inline tests/tests/test_image_update_failures.py:43]: Oh my... :see_no_evil: This was evil.

File: extra/mender-client-docker-addons/Dockerfile [hunk 3]
```diff
@@ -25,7 +27,19 @@ WORKDIR /src/mender
 RUN make prefix=/mender-install install
 RUN jq ".ServerCertificate=\"/usr/share/doc/mender-client/examples/demo.crt\" | .ServerURL=\"https://docker.mender.io/\"" \
     < examples/mender.conf.demo > /mender-install/etc/mender/mender.conf
-RUN echo artifact_name=original > /mender-install/etc/mender/artifact_info
+
+# TODO - Should this only handle the artifact_name in the Bootstrap Artifact, or checksum as well (?)
```
Comment (lluiscampos) [inline extra/mender-client-docker-addons/Dockerfile:31]: This client does not support rootfs-image updates, so `artifact_name` and  a dummy `rootfs-image.version` is enough :+1: 

Please clean-up the comment

File: extra/mender-client-docker-addons/Dockerfile [hunk 4]
```diff
@@ -6,8 +6,10 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update && apt-get install -y make git build-essential golang liblzma-dev \
     jq libssl-dev libglib2.0-dev curl
 
-ARG MENDER_CLIENT_REV=master
+# TODO - This cannot be hardcoded to anything else but master later
+ARG MENDER_CLIENT_REV=automatic_bootstrap_artifact
```
Comment (lluiscampos) [inline extra/mender-client-docker-addons/Dockerfile:10]: Remember to fix this

**Review by lluiscampos — CHANGES_REQUESTED** [review]
Overall looks good!

Few minor things and a concern in `test_update_modules.py` that we need to figure out.

File: extra/mender-client-docker-addons/Dockerfile (hunk 3 above)
Comment (oleorhagen) [reply extra/mender-client-docker-addons/Dockerfile:31]: good, thx!

File: tests/tests/test_basic_integration.py (hunk 1 above)
Comment (oleorhagen) [reply tests/tests/test_basic_integration.py:128]: I'll re-apply it.

File: tests/tests/test_image_update_failures.py (hunk 2 above)
Comment (oleorhagen) [reply tests/tests/test_image_update_failures.py:43]: hah, indeed!

File: extra/mender-client-docker-addons/Dockerfile (hunk 4 above)
Comment (oleorhagen) [reply extra/mender-client-docker-addons/Dockerfile:10]: This we will have to fix after the whole pipeline is green.

**Review by lluiscampos — APPROVED** [review]

**Comment by oleorhagen** [comment]
Build started.
Link: https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/662503123

**Comment by oleorhagen** [comment]
@mender-test-bot cherry-pick to: `3.3.x`

**Pull request merged**